XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -109,7 +109,7 @@ public bool SetupConfiguration(Type controllerType)
             {
                 // Assign any known interaction mappings.
                 if (controllerMappings[i].ControllerType?.Type == controllerType &&
-                    controllerMappings[i].Handedness == ControllerHandedness &&
+                    (controllerMappings[i].Handedness == ControllerHandedness || controllerMappings[i].Handedness == Handedness.Any) &&
```

Comment:
Both?

Reply:
Curious how I missed that, good spotting. Will double check and test.  However, doesn't explain why some of the interactions work.